extensions/amp-nexxtv-player/validator-amp-nexxtv-player.protoascii

```diff
@@ -50,12 +50,18 @@ tags: {  # <amp-nexxtv-player>
     }
   }
   attrs: {
-    name: "data-seek-to"
+    name: "data-delay"
   }
   attrs: {
     name: "data-streamtype"
     value_regex: "album|audio|live|playlist|playlist-marked|video"
   }
+  attrs: {
```

Comment:
`data-*` are always valid, no need for validator to list them (unless required or you wanna restrict the value)

Reply:
In my opinion it's best to start strict and then open up later if need be instead of being open and having to become strict later.